R/DiseasystoreBase.R

```diff
@@ -546,13 +558,13 @@ DiseasystoreBase <- R6::R6Class( # nolint: object_name_linter.
 
       # Determine the date ranges used
       logs <- logs |>
-        dplyr::mutate(fs_start_date = stringr::str_extract(message, "(?<=fs-range: )([0-9]{4}-[0-9]{2}-[0-9]{2})"),
-                      fs_end_date   = stringr::str_extract(message, "([0-9]{4}-[0-9]{2}-[0-9]{2})$")) |>
+        dplyr::mutate(fs_start_date = stringr::str_extract(.data$message, r"{(?<=fs-range: )(\d{4}-\d{2}-\d{2})}"),
+                      fs_end_date   = stringr::str_extract(.data$message, r"{(\d{4}-\d{2}-\d{2})$}")) |>
```

Comment:
Consider adding visible binding to `fs_start_date` and `fs_end_date` when applying the `dplyr::mutate()` function. 

I.e.

```
dplyr::mutate("fs_start_date" = stringr::str_extract(.data$message, r"{(?<=fs-range: )(\d{4}-\d{2}-\d{2})}"),
                      "fs_end_date"   = stringr::str_extract(.data$message, r"{(\d{4}-\d{2}-\d{2})$}")) |>
```